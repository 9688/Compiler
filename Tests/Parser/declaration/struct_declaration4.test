
struct{
	int x;
	char y;
	double i, j, k;
};