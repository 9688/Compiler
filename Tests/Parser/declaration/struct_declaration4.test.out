<--------TABLES-------->
<---TABLE--->
VARS::

TYPES::
char char

double double

int int

void void

